juju/entrypoint.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+# use modules from the lib directory - works under an assumption that
```

Comment:
Nitpick-level comment: so we can standardize the style of comments on the code base, let's please try to have proper English sentences, capitalized and punctuated as normal text would be. There are other instances of this below, but I won't repeat further.

Reply:
OK, we can agree on that.